aya/src/programs/tc.rs

```diff
@@ -118,25 +164,41 @@ impl SchedClassifier {
         load_program(BPF_PROG_TYPE_SCHED_CLS, &mut self.data)
     }
 
-    /// Attaches the program to the given `interface` using the default options.
+    /// Attaches the program to the given `interface` using the TCX link
```

Comment:
The doc format is:

[short first paragraph]
[newline]
[second longer paragraph]
[newline]
[maybe more paragraphs]

Periods and newlines are not optional :P

```
Attaches the program to the given `interface`.

On kernels >= X.Y.Z, it will attempt to use the TCX interface and attach as the first program. If TCX is not available, it will fallback to using the legacy netlink interface.

For finer grained control over the attach options, see [SchedClassifier::attach_with_options].
```

Reply:
again sorry about that, fixed!